cob_bringup/robots/cob4-1.launch

```diff
@@ -5,17 +5,17 @@
 	<arg name="pkg_env_config" default="$(find cob_default_env_config)"/>
```

Comment:
Well, the navigation does not rely on this file at all, afaik.
This file is used by the dashboard to specify the goals, which are then sent via scriptserver to the navigation.
I don't know of any other case, but maybe it is good to listen to @ipa-srd and @ipa-flg as well.